### PR TITLE
Change how `setCellAttr` check multiple cells

### PR DESCRIPTION
**Pull request by dkozickis**
* If `CellSelection` has multiple cells selected, check all of those cells.
* If single cell is selected - perform normal check

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -568,7 +568,18 @@ export function setCellAttr(name: string, value: unknown): Command {
...
-    if ($cell.nodeAfter!.attrs[name] === value) return false;
+
+    const isMultipleCellSelection =
+      state.selection instanceof CellSelection &&
+      state.selection.isMultipleCellSelection();
+
+    // If selection is multiple cell selection, don't bail early
+    // this allows to set the attribute to the same value for all cells, even
+    // if some cells already have the attribute set to that value
+    if (!isMultipleCellSelection && $cell.nodeAfter!.attrs[name] === value) {
```
Comment (piemonkey) [inline src/commands.ts]: Hmm, don't we want to go through each of the cells and bail if all of them have the attribute? For example, we shouldn't be able to clear the color of a selection of cells that all have no color. I think this version is okay if that's hard or fiddly though.

**Review by elpoelma — COMMENTED** [review]
Looks good to me, just wondering: if cell-selection encompassed multiple cells, could it be useful to return `false` if all the cells in this selection already contain the attribute with the provided value?

**Comment by dkozickis** [comment]
Y'all got me there :D

**Review by piemonkey — APPROVED** [review]
The Set approach caught me by surprise, but it works.

**Pull request merged**